### DISCUSSION TITLE
Ajustes em limite de CPS + Punições

### DISCUSSION THREAD

**Discussion by Batatoones** [leogianfagna/IssueTracker#921] [Opinioes]
### Descrição da tarefa

Ideia é aumentar o limite de CPS em todos os server seja evento, henesys, vanillew pois a galera vem pedindo e isso iria acompanhar algumas mudanças como:
- Limite novo de CPS é 20 CPS
- Punição nova: passou de 20 CPS toma kick na primeira vez e depois começa a tomar banimento
- Quem sabe alterar o barulho do hit para evitar reclamações de barulho de CPS (não é obrigatório)

**Comment by leogianfagna** [comment]
@Batatoones acho mais válido pedir pra um player abrir sugestão disso no dc ao invés da gente tomar a decisão sozinho. sobre o som, sou a favor independente do que mude

**Comment by leogianfagna** [comment]
(só não sei como fazer)
